build.env

```diff
@@ -12,8 +12,8 @@
 CSI_IMAGE_VERSION=canary
 
 # Ceph version to use
-BASE_IMAGE=docker.io/ceph/ceph:v15
-CEPH_VERSION=octopus
+BASE_IMAGE=docker.io/ceph/ceph:v16
+CEPH_VERSION=pacific
```

Comment:
This also requires a go-ceph version that has support for Pacific. If you want to have the features from rbd-nbd, you do not need to update the `CEPH_VERSION`, as that is only used for go-ceph at the moment.

Reply:
@pkalever can we just update the base image, not the CEPH_VERSION, go-ceph is not currently supporting the pacific release https://github.com/ceph/go-ceph#supported-ceph-versions

Reply:
Oops! for some reason I thought this PR is merged.
Apologies for delaying, will make an update right now.

Thanks!

Reply:
@Madhu-1 done now.